reference/_guide-to-comp-typography.md

```diff
@@ -0,0 +1,93 @@
+# GUIDE TO COMP TYPOGRAPHY
```

Comment:
See my previous comments on Markdown formatting and code samples.

Also, why the underscore in the name of this file?